standards/nuts.rst

```diff
@@ -0,0 +1,30 @@
+.. _nuts:
+
+NUTS classification
+===================
+
+The :py:mod:`nomenclature` (`read the docs`_) package makes use of the :py:mod:`pysquirrel` package
+(`link <https://github.com/iiasa/pysquirrel>`_) to provide a utility for region
+names based on the `NUTS classification <https://ec.europa.eu/eurostat/web/nuts>`_. 
+
+This feature allows users to define an agreed list of territorial units with 
```

Comment:
The users don't really define the list. I'd change it to
```suggestion
This feature allows users to easily import the NUTS regions which are territorial units with 
```
or something along those lines.